components/table/demo/size.md

```diff
@@ -1,8 +1,16 @@
 ---
 order: 9
-title: 紧凑型
+title:
+  en-US: size
+  zh-CN: 紧凑型
 ---
 
+## en-US
+
+Compacted table size: `middle` and `small`, `small` size is used in Modal only.
```

Comment:
Compact table size: `middle` and `small`, `small` size is used in Modal only, and `middle` is recommended to present compact data